SWLOR.Game.Server/Feature/ChatCommandDefinition/DMChatCommand.cs

```diff
@@ -406,10 +406,66 @@ private void SetLocalVariable()
 
                     SendMessageToPC(user, "Local string set: " + variableName + " = " + value);
                 });
+
+            _builder.Create("settag")
+                .Description("Sets a local tag on a target.")
+                .Permissions(AuthorizationLevel.DM, AuthorizationLevel.Admin)
+                .AvailableToAllOnTestEnvironment()
+                .RequiresTarget()
+                .Validate((user, args) =>
+                {
+                    if (args.Length <= 0)
+                    {
+                        return "Missing arguments. Format should be: /SetTag <VALUE>. Example: /SetTag DUNGEON_ENTRANCE";
+                    }
+
+                    return string.Empty;
+                })
+                .Action((user, target, location, args) =>
+                {
+                    if (!GetIsObjectValid(target))
+                    {
+                        SendMessageToPC(user, "Target is invalid. Please target a creature, placeable or door.");
+                    }
+                    else
+                    {
+                        SetTag(target, args[0]);
+
+                        SendMessageToPC(user, "Tag set to: " + args[0] + ".");
+                    }
+                });
+
+            _builder.Create("destination")
+                .Description("Sets the DESTINATION localstring to the given value on a target.")
+                .Permissions(AuthorizationLevel.DM, AuthorizationLevel.Admin)
+                .AvailableToAllOnTestEnvironment()
+                .RequiresTarget()
+                .Validate((user, args) =>
+                {
+                    if (args.Length <= 0)
+                    {
+                        return "Missing arguments. Format should be: /destionation <VALUE>. Example: /destination EventEntrance";
+                    }
+
+                    return string.Empty;
+                })
+                .Action((user, target, location, args) =>
+                {
+                    if (!GetIsObjectValid(target))
+                    {
+                        SendMessageToPC(user, "Target is invalid. Please target a placeable.");
```

Comment:
This is misleading because it will actually work on any type of object so long as it's valid. This means you could target items, players, etc. If you really intend this to be used on placeables, please change the if statement to do a check on GetObjectType instead.